build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -4,12 +4,26 @@
   <PropertyGroup>
     <_SourceTopDir>..\..</_SourceTopDir>
     <_BclFrameworkDir>$(OutputPath)\lib\xbuild-frameworks\MonoAndroid\v1.0</_BclFrameworkDir>
+    <_MandroidDir>$(OutputPath)\lib\mandroid</_MandroidDir>
   </PropertyGroup>
+  <ItemGroup>
+    <MonoDocItem Include="monodoc.dll" />
+    <MonoDocItem Include="monodoc.dll.mdb" />
+    <MonoDocItem Include="monodoc.dll.config" />
```

Comment:
mdoc.exe is special, because it is remapped and not copied. I have renamed the groups to MonoDocCopyItem and MonoDocCopyItems to hopefully make it more clear.